inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/UserServiceImpl.java

```diff
@@ -90,6 +92,9 @@ public boolean create(UserInfo userInfo) {
         entity.setDueDate(getOverDueDate(userInfo.getValidDays()));
         entity.setCreateBy(LoginUserUtils.getLoginUserDetail().getUserName());
         entity.setName(username);
+        entity.setSecretKey(userInfo.getSecretKey());
```

Comment:
Maybe we should generate the secret key, public key, and private key during saving user info.

Reply:
only those who transfer data through public network needs it.